### ISSUE TITLE
Uncaught exception - [lime.utils.Preloader] ERROR: There is no asset library with an ID of "default" at startup.

### ISSUE THREAD

**Issue by nennneko5787** [HaxeFlixel/flixel#2972]
- **Haxe version:** 4.3.2
- **Flixel version:** 5.5.0
- **OpenFL version:** 9.3.2
- **Lime version:** 8.1.1
- **Affected targets:** neko, windows

________________________________

**Code snippet reproducing the issue**:

```haxe
package;

import flixel.FlxState;

class PlayState extends FlxState
{
	override public function create()
	{
		super.create();
	}

	override public function update(elapsed:Float)
	{
		super.update(elapsed);
	}
}
```

________________________________

**Observed behavior:**
Uncaught exception - [lime.utils.Preloader] ERROR: There is no asset library with an ID of "default"
![image](https://github.com/HaxeFlixel/flixel/assets/95109758/457bc294-e75b-4f1d-89a6-4efa601cdc50)

**Expected behavior:**
Normal startup.

**Comment by Geokureli** [comment]
what does your project.xml look like

**Comment by nennneko5787** [comment]
I dropped the Flixel version and it solved the problem.

**Comment by nennneko5787** [comment]
Sorry for the delay.
I got an error in the project.xml generated by the flixel tpl command
```xml
<?xml version="1.0" encoding="utf-8"?>
<project xmlns="http://lime.software/project/1.0.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xsi:schemaLocation="http://lime.software/project/1.0.2 http://lime.software/xsd/project-1.0.2.xsd">

	<!-- _________________________ Application Settings _________________________ -->

	<app title="TickTick" file="TickTick" main="Main" version="0.0.1" company="HaxeFlixel" />

	<!--The flixel preloader is not accurate in Chrome. You can use it regularly if you embed the swf into a html file
		or you can set the actual size of your file manually at "FlxPreloaderBase-onUpdate-bytesTotal"-->
	<app preloader="flixel.system.FlxPreloader" />

	<!--Minimum without FLX_NO_GAMEPAD: 11.8, without FLX_NO_NATIVE_CURSOR: 11.2-->
	<set name="SWF_VERSION" value="11.8" />

	<!-- ____________________________ Window Settings ___________________________ -->

	<!--These window settings apply to all targets-->
	<window width="640" height="480" fps="60" background="#000000" hardware="true" vsync="false" />

	<!--HTML5-specific-->
	<window if="html5" resizable="false" />

	<!--Desktop-specific-->
	<window if="desktop" orientation="landscape" fullscreen="false" resizable="true" />

	<!--Mobile-specific-->
	<window if="mobile" orientation="landscape" fullscreen="true" width="0" height="0" />

	<!-- _____________________________ Path Settings ____________________________ -->

	<set name="BUILD_DIR" value="export" />
	<source path="source" />
	<assets path="assets" />

	<!-- _______________________________ Libraries ______________________________ -->

	<haxelib name="flixel" />

	<!--In case you want to use the addons package-->
	<!--<haxelib name="flixel-addons" />-->

	<!--In case you want to use the ui package-->
	<!--<haxelib name="flixel-ui" />-->

	<!--In case you want to use nape with flixel-->
	<!--<haxelib name="nape-haxe4" />-->

	<!-- ______________________________ Haxedefines _____________________________ -->

	<!--Enable the Flixel core recording system-->
	<!--<haxedef name="FLX_RECORD" />-->

	<!--Disable the right and middle mouse buttons-->
	<!--<haxedef name="FLX_NO_MOUSE_ADVANCED" />-->

	<!--Disable the native cursor API on Flash-->
	<!--<haxedef name="FLX_NO_NATIVE_CURSOR" />-->

	<!--Optimise inputs, be careful you will get null errors if you don't use conditionals in your game-->
	<haxedef name="FLX_NO_MOUSE" if="mobile" />
	<haxedef name="FLX_NO_KEYBOARD" if="mobile" />
	<haxedef name="FLX_NO_TOUCH" if="desktop" />
	<!--<haxedef name="FLX_NO_GAMEPAD" />-->

	<!--Disable the Flixel core sound tray-->
	<!--<haxedef name="FLX_NO_SOUND_TRAY" />-->

	<!--Disable the Flixel sound management code-->
	<!--<haxedef name="FLX_NO_SOUND_SYSTEM" />-->

	<!--Disable the Flixel core focus lost screen-->
	<!--<haxedef name="FLX_NO_FOCUS_LOST_SCREEN" />-->

	<!--Disable the Flixel core debugger. Automatically gets set whenever you compile in release mode!-->
	<haxedef name="FLX_NO_DEBUG" unless="debug" />

	<!--Enable this for Nape release builds for a serious peformance improvement-->
	<haxedef name="NAPE_RELEASE_BUILD" unless="debug" />

	<!-- _________________________________ Custom _______________________________ -->

	<!--Place custom nodes like icons here (higher priority to override the HaxeFlixel icon)-->
</project>

```

**Comment by UncertainProd** [comment]
@nennneko5787 What is the path to your exe file? I had a [similar issue](https://github.com/openfl/lime/issues/1725) on a lime-only project and it turns out that the version of SDL that lime uses in v8.1 doesn't handle long file paths (> 126 characters) very well. For me, however, the fix was to upgrade to lime 8.2 which uses an updated version of SDL
